src/main/java/org/humancellatlas/ingest/file/web/FileController.java

```diff
@@ -84,4 +103,94 @@
         PersistentEntityResource resource = assembler.toFullResource(updatedFile);
         return ResponseEntity.accepted().body(resource);
     }
+
+    @RequestMapping(path = "/files/{id}/inputToProcesses", method = {PUT, POST}, consumes = {TEXT_URI_LIST_VALUE})
```

Comment:
Could you DRY the above methods, please? I believe you can even dry it with the other controllers you changed in this PR. Tx